content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -82,10 +85,15 @@ decided by the payload of webhook also.
 
 === How to run the Plugin
 
-See link:https://github.com/jenkinsci/simple-pull-request-job-plugin/blob/master/README.md[How to run the demo]
+See link:https://github.com/Jenkinsci/simple-pull-request-job-plugin/blob/master/README.md[How to run the demo]
 and set credentials, owner and repository on your own and you will be good to go.
 
-=== What is avalilable now
+==== Example branch-source configuration.
+image:/images/post-images/gsoc-simple-pull-request-plugin/branch-source-configuration.png[title="Jenkinsfile.yaml", role="center"]
+
+
+
+=== What is available now
 
 . Users are able to select to use Jenkinsfile.yaml for build.
```

Comment:
Users are able to select the `Jenkinsfile.yaml` file as the source for the Pipeline configuration.